### PR TITLE
fix styles

### PR DESCRIPTION
**Pull request by ZhraaBaharAluloom**
Fixed update book details, contact and edit image modals

### CODE REVIEW

File: src/components/ContactModal.jsx [hunk 1]
```diff
@@ -52,7 +52,7 @@ function ContactModal({
               duration: 0.5,
             },
           }}
-          className="flex flex-col justify-center items-center text-black bg-white sm:w-96 rounded-lg p-10 mx-auto relative w-80 font-sans text-sm sm:text-lg"
+          className="flex flex-col  items-center    text-primary bg-black rounded-lg p-10  relative font-sans md:w-96 text-sm sm:text-lg"
```
Comment (RawandDev) [inline src/components/ContactModal.jsx]: Not sure if this is something important to mention but I see some extra double and triple spaces which is not necessary.
Before `text-primary` there are triple spaces and some other places there are two.

File: src/components/EditImageModal.jsx [hunk 2]
```diff
@@ -48,72 +48,72 @@ function EditImageModal({ isEditImageOpen, setIsEditImageOpen }) {
...
                   src={editIcon}
                   alt=""
-                  className="bg-black w-6 mb-4 rounded-full cursor-pointer"
+                  className=" w-6 mb-4 rounded-full cursor-pointer"
                 />
                 <img
                   src={trashIcon}
                   alt=""
-                  className="bg-black rounded-full cursor-pointer"
+                  // className="bg-primary rounded-full cursor-pointer"
```
Comment (RawandDev) [inline src/components/EditImageModal.jsx]: There are three comments. Can you remove them?

**Review by RawandDev — CHANGES_REQUESTED** [review]
Well done Zahra, They look really nice.
I have added two comments to improve a bit.

**Review by MuhannedNoman — APPROVED** [review]
Feel free to merge after seeing Rawand's comments.

File: src/components/ContactModal.jsx (hunk 1 above)
Comment (ZhraaBaharAluloom) [reply src/components/ContactModal.jsx]: > Not sure if this is something important to mention but I see some extra double and triple spaces which is not necessary. Before `text-primary` there are triple spaces and some other places there are two.

No it's not sth important to mention 🙂, but i'll do it

**Review by RawandDev — APPROVED** [review]
LGTM

**Pull request merged**